pages/children.md

```diff
@@ -10,10 +10,9 @@ Our day for children is back!
 We're hosting a fun-packed day of activities and workshops for children to
 learn all about getting the most out of the world with Python.
 
-We're working with the Raspberry Pi Foundation, and some of the team who
-brought Python to the Micro:bit will be coming along to show you how use Python
-to write games.  There'll be a chance to play with robots, and to learn how to
-program Minecraft with Python!
+This year we're working with the Raspberry Pi Foundation, Code Club and some of the team who
+brought Python to the BBC Micro:bit. They will be coming along to show you how use Python
```

Comment:
Nitpick: the BBC computer is stylised either "Micro Bit" or "micro:bit".

Also, I missed this in the original: should it be "how _to_ use Python to write"?